tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -57,4 +58,19 @@ describe('<ShowMoreCard />', () => {
     const rootNode = findDOMNode(root);
     expect(rootNode.textContent).toContain('Hello I am description');
   });
+
+  it('executes truncateToMaxHeight when it recieves props', () => {
+    const node = document.createElement('div');
+    let ref;
+    ReactDOM.render(<ShowMoreCardBase i18n={getFakeI18nInst()} ref={(e) => { ref = e; }} />, node,
```

Comment:
This test is a bit too fragile. Testing the `componentWillReceiveProps()` method is easier with [enzyme](https://github.com/airbnb/enzyme):

```js
import { shallow } from 'enzyme';

const root = shallow(<ShowMoreCardBase i18n={getFakeI18nInst()} />);
const component = root.instance();

const contentNode = ReactDOM.findDOMNode(instance.contents);
const truncateToMaxHeight = sinon.spy(component, 'truncateToMaxHeight');
root.setProps(); // simulate any kind of update to properties

sinon.assert.calledWith(truncateToMaxHeight, contentNode);
```

We already have enzyme but this suite hasn't been converted yet. Try the code above and let me know if it works.